Resources/Private/JavaScript/asset-editing/src/components/EditAssetDialog.tsx

```diff
@@ -67,21 +67,21 @@ const EditAssetDialog: React.FC = () => {
     return (
         <Dialog
             isOpen={dialogVisible}
-            title={translate('EditAssetDialog.title', 'Rename asset')}
+            title={translate('editAssetDialog.title', 'Rename asset')}
```

Comment:
My idea about using uppercase for component names in xlf was that we will have a lot of labels in one file (until  we find a good way to split this) and that it might make it easier to distinguish between components and general labels. Didn't discuss this with anyone yet though.

Reply:
I laways go the same naming way and if i want to know where belongs some translation i either use some prefix in its name or add comment in xlf file. I also always just search files for given translation place i i want to check anything.